PYTHON_BINDINGS.md

```diff
@@ -0,0 +1,15 @@
+Python Bindings
+---------------
+
+The Python app is located under: decArgo_soft/soft/decoder_api
+
+
+To run the decoder with its associated bindings, run the following steps.
+
+
+1. Build the image: `docker build -f Dockerfile --target python-runtime -t  decoder_matlab_tool:latest .`
+
+2. Run the containers: `docker compose -f .\docker-compose.yml up -d --build`
+
+The logs and files are written to the area: decArgo_demo/output.
```

Comment:
As I mentioned yesterday I think it may not be immediately obvious that the command to run the decoder is hardcoded into the Docker image, so perhaps a Python script that shows some example usage of the bindings could make this clearer or explicitly mentioning which commands are being run by what in this example.

We probably also want to explain where the input files are being taken from that are being decoded in this example.